.travis.yml

```diff
@@ -1,18 +1,20 @@
 language: objective-c
 osx_image: xcode8.2
 before_install:
-- scripts/import_keychain.sh
-- security find-identity -v -p codesigning
-- gem install pact-mock_service -v 0.12.0
-- bundle install
-- carthage checkout
+  - scripts/import_keychain.sh
+  - security find-identity -v -p codesigning
+  - gem install pact-mock_service -v 0.12.0
+  - bundle install
+  - scripts/install_swiftlint.sh
```

Comment:
Could swiftlint be installed via homebrew? (brew should be installed on travisci). Might be simpler to maintain....